build.gradle

```diff
@@ -20,7 +20,7 @@ ext {
     automatticRestVersion = '1.0.8'
     automatticStoriesVersion = '2.1.0'
     automatticTracksVersion = '3.0.0'
-    gutenbergMobileVersion = 'v1.97.0-alpha1'
+    gutenbergMobileVersion = '5849-6b3b4f2daf124e06d07256aacf342be6488c66eb'
```

Comment:
A friendly reminder that we will need to update that this to the `v1.97.0` tag once it is published.